modular_skyrat/modules/colony_fabricator/code/tools/tools.dm

```diff
@@ -0,0 +1,150 @@
+// Like the power drill, except no speed buff but has wirecutters as well? Just trust me on this one.
+
+/obj/item/screwdriver/omni_drill
+	name = "powered driver"
+	desc = "The ultimate in multi purpose construction tools. With heads for wire cutting, bolt driving, and driving \
+		screws, what's not to love? Well, the slow speed. Compared to other power drills these tend to be \
+		<b>not much quicker than unpowered tools</b>."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/tools.dmi'
+	icon_state = "drill"
+	belt_icon_state = null
+	inhand_icon_state = "drill"
+	worn_icon_state = "drill"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	custom_materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+		/datum/material/silver = HALF_SHEET_MATERIAL_AMOUNT * 1.5,
+		/datum/material/titanium = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	force = 10
+	throwforce = 8
+	throw_speed = 2
+	throw_range = 3
+	attack_verb_continuous = list("drills", "screws", "jabs", "whacks")
+	attack_verb_simple = list("drill", "screw", "jab", "whack")
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	w_class = WEIGHT_CLASS_SMALL
+	toolspeed = 1
```

Comment:
this should probably be like 1.2/1.3 to make up for it replacing 3 tools (also because it's not actually slow, that's just normal toolspeed)